app/views/layouts/application.html.erb

```diff
@@ -9,8 +9,10 @@
 
     <%= stylesheet_link_tag "https://www-media.stanford.edu/su-identity/css/su-identity.css" %>
     <%= stylesheet_link_tag "https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700" %>
+    <%= stylesheet_link_tag "https://cdn.jsdelivr.net/npm/simple-datatables@latest/dist/style.css" %>
     <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <script src="https://cdn.jsdelivr.net/npm/simple-datatables@latest"></script>
```

Comment:
Why not just put this in the component itself?

Reply:
That makes sense, I wasn't sure if we wanted this available elsewhere, but it probably makes the most sense to only include it when we need it.